tests/setup/after-env.js

```diff
@@ -2,9 +2,12 @@ import { Wrapper, WrapperArray } from '@vue/test-utils';
 
 require('snapshot-diff/extend-expect');
 
+const findAllByDataTest = function findAllByDataTest(dataTest) {
```

Comment:
you can remove `const findAllByDataTest = ` bit. You declare named function, you can use it directly.